settings.gradle

```diff
@@ -1,6 +1,6 @@
 
-rootProject.name = 'javapfb'
+rootProject.name = 'java-pfb'
 include('library', 'cli')
 
-gradle.ext.releaseVersion = '0.3.0'
-
+gradle.ext.projectGroup = 'bio.terra'
+gradle.ext.releaseVersion = '0.3.0-SNAPSHOT'
```

Comment:
I'm not sure if the bumper GHA will still work with the "-SNAPSHOT" suffix. However, it's required for submitting to artifactory. So, I could alternatively append this in the library publishing.gradle.

Reply:
I think that's only true if the library is published as a snapshot. If you publish it as a release then it doesn't need `-SNAPSHOT`. For example, the catalog client library doesn't have `SNAPSHOT` in its version and it's published to artifactory. https://broadinstitute.jfrog.io/ui/repos/tree/General/libs-release-local/bio/terra/catalog-client

Reply:
Ah, ok! That makes a lot of sense. I've removed "-SNAPSHOT" and am now pointing to libs-release-local on jfrog.